blog/tutorial/01-index.md

```diff
@@ -186,15 +194,15 @@ func (msg MsgCreatePost) ValidateBasic() error {
   }
   return nil
 }
-``` -->
+```
 
 Going back to `GetCmdCreatePost` in `x/blog/client/cli/tx.go`, you'll see `MsgCreatePost` being created and broadcast with `GenerateOrBroadcastMsgs`.
 
 After being broadcast, the messages are processed by an important part of the application, called [**handlers**](https://docs.cosmos.network/master/building-modules/handler.html).
 
 ## x/blog/handler.go
 
-Begin by importing your new blog types that we created:
+Begin by checking if the new blog types that we created is imported:
 
 ```go
```

Comment:
```go
import (
  // Existing imports...
  "github.com/example/blog/x/blog/types"
)
```

@Tosch110 can you please double-check, but I believe the default scaffold already imports `types` package, so if a user includes this import (and their editor isn't smart), Starport throws an error.

Reply:
@fadeev this is correct, that's why I changed the wording above the sentence. Do you think we should rather erase this part then?

Reply:
You're right, my bad 🙂